libexec/auto_downtimes.py

```diff
@@ -207,7 +207,7 @@ def egi_end_element(name):
     try:
       new_dt_file.write("%s\n" % desc_str)
     except UnicodeEncodeError as ue:
-      print "Unicode not allowed; skipping description: %s: %s" % (desc_str, ue)
+      print ("Unicode not allowed; skipping description: %s: %s" % (desc_str, ue)).encode('ascii', 'ignore').decode('ascii')
```

Comment:
Since we can't print the character it is failing on anyway, as an alternative, let's just not print the description in the error message:

```suggestion
      print "Unicode not allowed; skipping description for %s: %s" % (entry, ue)
```